### DISCUSSION TITLE
In the dark theme, background transparency does not work. In the light theme, it works.

### DISCUSSION THREAD

**Discussion by tebaly** [tailwindlabs/tailwindcss#14226] [Help]
In the dark theme, background transparency does not work. In the light theme, it works.

```class=" bg-black dark:bg-zinc-800 bg-opacity-50"```

**What version of Tailwind CSS are you using?**

  ^3.4.10

**What build tool (or framework if it abstracts the build tool) are you using?**

"nuxt": "^3.7.4",
"tailwindcss": "^3.4.10",
"@nuxtjs/tailwindcss": "^6.12.1",

**What version of Node.js are you using?**

For example: v18 

**What browser are you using?**

For example: Chrome 

**What operating system are you using?**

For example: Linux

**Comment by adamwathan** [comment]
Hey! What you're seeing is that the `dark:bg-zinc-800` class resets the opacity back to 100%, so you need to respecify it in dark mode with `dark:bg-opacity-50` or just use `bg-black/50 dark:bg-zinc-800/50`.
